rca/static_src/javascript/programmes/components/ProgrammesExplorer.js

```diff
@@ -17,7 +17,8 @@ const ProgrammesExplorer = ({ searchLabel, categories }) => {
     const loc = useLocation();
     const params = new URLSearchParams(loc.search);
     const activeCategory = params.get('category') || categories[0].id;
-    const activeValue = params.get('value');
+    const filterValue = params.get('value') || '';
+    const activeValue = filterValue.split('-')[0];
```

Comment:
I have checked that this does not explode if splitting an empty string, or splitting a string that has no `-` – it’s safe 😅.